src/client/pages/Offer/Introduction/Sidebar/SwitchingNotice.tsx

```diff
@@ -18,7 +19,9 @@ export const SwitchingNotice = () => {
       </Notice.Header>
       <ReactMarkdown source={textKeys.SIDEBAR_SWITCHING_NOTICE_BODY()} />
 
-      <a href="#">{textKeys.SIDEBAR_SWITCHING_NOTICE_LINK()}</a>
+      <a href={PromotionTypes.SWITCH}>
```

Comment:
Just curious, how href="promotion-switch" here would translate to #promotion-switch on the page?

Reply:
It won't, it's a mistake from my part! Nice catch! 🙏